inject/src/main/java/io/micronaut/context/ApplicationContextBuilder.java

```diff
@@ -36,6 +36,27 @@
  */
 public interface ApplicationContextBuilder {
 
+    /**
+     * Sets the trace mode for bean resolution.
+     * @param traceMode The debug mode
+     * @since 4.8.0
+     * @see BeanResolutionTraceMode
+     * @return This builder
+     */
+    default @NonNull ApplicationContextBuilder traceMode(@NonNull BeanResolutionTraceMode traceMode) {
```

Comment:
Can this be `beanResolutionTrace(@NonNull BeanResolutionTraceMode traceMode, String... classPatterns)`